fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/TemplateDruidQuery.java

```diff
@@ -196,6 +202,9 @@ public boolean isTimeGrainValid() {
     public TemplateDruidQuery merge(TemplateDruidQuery sibling) {
 
         // TODO: Handle merging with a null TDQ
+//        if (!nestedQuery.isPresent()) {
```

Comment:
If we don't need this code, we should just delete it. Let version control remember it.